great_expectations/rule_based_profiler/helpers/util.py

```diff
@@ -620,6 +627,17 @@ def compute_kde_quantiles_point_estimate(
     lower_quantile_pct: float = false_positive_rate / 2.0
     upper_quantile_pct: float = 1.0 - (false_positive_rate / 2.0)
 
+    ndarray_is_datetime_type: bool = is_ndarray_datetime_dtype(
+        data=metric_values, parse_strings_as_datetimes=True
+    )
+
+    metric_values_original: np.ndarray
```

Comment:
Why not `MetricValues`?

Reply:
@NathanFarmer Good question -- I have been keeping that one for N-dimensional arrays (we will need to modernize this scheme at some point; not urgent now).  Thanks.